### PR TITLE
Fixes and improvements for processing GenomeScan samples

### PR DESCRIPTION
**Pull request by pneerincx**
* ```processGsRawData.sh```:
  * Improved sanity check.
  * Allow multiple lanes per sample potentially from multiple flowcells.
  * Removed blank lines.
  * Updated for changed commandline args in ```createInhouseSamplesheetFromGS.py```.
  * Changed format of control file created when the script is finished to make it in sync with the one demultiplexing.sh creates and which copyRawDataToPrm.sh expects.
* ```createInhouseSamplesheetFromGS.py``` major rewrite:
  * Improved sanity check.
  * Allow mutiple lanes per sample potentially from multiple flowcells
  * Standardized variable naming consistent with other scripts from this same repo
  * Simplified nested data structures by removing unused levels of nesting.
  *  Use standard Python logging facility as opposed to writing a custom log file.
* ```copyRawDataToPrm.sh```:
  * Removed non-working email commandline option
  * Added commandline option to specify an alternative ```SCR_ROOT_DIR``` to use as root on the source server.
* ```boxy.umcg-gd-dm.cron```: Updated for new version numbers and option to specify an alternative ```SCR_ROOT_DIR``` to use as root on the source server for ```copyRawDataToPrm.sh```.

### CODE REVIEW

File: bin/copyRawDataToPrm.sh [hunk 1]
```diff
@@ -260,7 +249,7 @@ function splitSamplesheetPerProject() {
 	local _rsyncControlFileFinished="${PRM_ROOT_DIR}/logs/${_run}/run01.${SCRIPT_NAME}.finished"
 	local _controlFileBase="${PRM_ROOT_DIR}/logs/${_run}/run01.splitSamplesheetPerProject"
 	local _logFile="${_controlFileBase}.log"
-	
+	#
```
Comment (RoanKanninga) [inline bin/copyRawDataToPrm.sh]: Why would you have only a hashtag when it is not  a comment?

**Comment by pneerincx** [comment]
> You have to add the public key of ATEAMBOT to the DATA_MANAGER user then since the script is executed by the data manager

I've reverted that change. Adding the key is one thing, but when pulling data from de umcg-genomescan group into the umcg-gd group using copyRawDataToPrm.sh, you would also have to switch to the ATEAMBOT of that other group, which is currently not possible. We could add some if then else logic, but it makes the setup of which keys have to be added to which user rather complex  :o

File: bin/copyRawDataToPrm.sh (hunk 1 above)
Comment (pneerincx) [reply bin/copyRawDataToPrm.sh]: Silly workaround for loosing indentation of empty lines by others using other editors. I've removed all indented blank lines and replaced them with comments. Most of them with some additional text, a few without; those are just to separate two blocks of code.

File: bin/processGsRawData.sh [hunk 2]
```diff
@@ -463,23 +464,21 @@ function processSamplesheetsAndMoveCovertedData() {
...
 	createInhouseSamplesheetFromGS.py \
-		--GenomeScanInputDir "${TMP_ROOT_DIR}/${_batch}/" \
-		--logfile "${_logFile}" \
-		--samplesheetNewDir "${TMP_ROOT_DIR}/Samplesheets/new/" \
-		--samplesheetOutputDir "${TMP_ROOT_DIR}/${_batch}/" \
+		--genomeScanInputDir "${TMP_ROOT_DIR}/${_batch}/" \
+		--inhouseSamplesheetsInputDir "${TMP_ROOT_DIR}/Samplesheets/new/" \
+		--samplesheetsOutputDir "${TMP_ROOT_DIR}/${_batch}/" \
+		--logFile "${_logFile}" \
 		>> "${_logFile}" 2>&1 \
```
Comment (RoanKanninga) [inline bin/processGsRawData.sh]: how does this work? You are having logFile as output file and also redirecting to that same logFile?

Comment (pneerincx) [reply bin/processGsRawData.sh]: Good point. This was unchanged as compared to the previous version (only indentation of changed), and it did not cause problems previously, but it's a bit silly. I suggest to remove the --logFile option for processGsRawData.sh and simply write any status messages to STDOUT. If you want to log, you can always redirect to a log file. Is that Ok?

Comment (RoanKanninga) [reply bin/processGsRawData.sh]: yep ,sounds good to me

Comment (RoanKanninga) [reply bin/processGsRawData.sh]: reminder :)

Comment (pneerincx) [reply bin/processGsRawData.sh]: Check, updated in last commit...

**Pull request merged**